packages/runtime/src/helpers/functions.ts

```diff
@@ -165,18 +165,22 @@ export const setupImageFunction = async ({
 }
 
 /**
- * Look for API routes, and extract the config from the source file.
+ * Look for extended API routes (background and scheduled functions) and extract the config from the source file.
  */
-export const getApiRouteConfigs = async (publish: string, baseDir: string): Promise<Array<ApiRouteConfig>> => {
+export const getExtendedApiRouteConfigs = async (publish: string, baseDir: string): Promise<Array<ApiRouteConfig>> => {
```

Comment:
[pebble] See the other comment: I think this shoudl remain as it is, and you could create a separate `getExtendedApiRouteConfigs` function that calls and filters `getApiRouteConfigs`.

Reply:
Disregard my last comment above. I'll have a separate function like you suggested to get the filtered extend API route configs.